checkpoint-multi-nic/azuredeploy.parameters.json

```diff
@@ -1,27 +1,39 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "adminPassword": {
-            "value": "changeme"
-        },
-        "authenticationType": {
-            "value": "password"
-        },
-        "newStorageAccountName": {
-            "value": "changeme"
-        },
-        "offer": {
-            "value": "R7710BYOL"
-        },
-        "sshPublicKey": {
-            "value": ""
-        },
-        "vmName": {
-            "value": "changeme"
-        },
-        "vmSize": {
-            "value": "Standard_D2"
-        }
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
This should be "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#"